### PR TITLE
Adds API documentation site

### PR DESCRIPTION
**Pull request by mchurichi**
Adds auto-generated API documentation site

image

Signed-off-by: Maximiliano Churichi <mchurichi@gmail.com>

### CODE REVIEW

**Comment by mchurichi** [comment]
@wibarre you may find this useful

**Comment by n-th** [comment]
Thanks, Maxi. Can you add instructions on how to run the doc server?

**Review by n-th — APPROVED** [review]

**Review by jufantozzi — APPROVED** [review]
💯 !!

File: Makefile [hunk 1]
```diff
@@ -156,4 +156,13 @@ $(eval $(call binary_rule,bin/spire-bridge-server,./cmd/jwtglue))
 # #		--go-spire_opt=mode=plugin \
 # #		$<
 
+dev-api-build:
+	docker build -f dev/api/Dockerfile -t galadriel-dev-api:latest .
+
+dev-api: dev-api-build
```
[the remark this answers is not in the document]
Comment (mchurichi) [reply Makefile:162]: `api-doc` sounds better 💯

**Comment by mchurichi** [comment]
> 

yes, sounds reasonable, I'll force-push this

**Pull request merged**